calc/__init__.py

```diff
@@ -6,4 +6,4 @@
 
 from .calc import *
 
-__version__ = '2.0.1'
+__version__ = '3.0.0'
```

Comment:
Thanks for the version bump.

Remember that upon merging this and [creating a new package release](https://github.com/openedx/openedx-calc/releases), edx-platform will automatically upgrade to the new version of this library, even though the old symmath hasn't been removed from edx-platform.  That may not be what you want. I recommend a [solution here](https://github.com/openedx/openedx-calc/pull/38#issuecomment-1013439750).